doc/y_cable/design_doc.md

```diff
@@ -80,19 +98,31 @@ Vendors can have several implementations/ways to use this concept
   - Currently the assumption is Y-cable API will be written in the Python language
   - If a vendor has an existing library in a different language, they will need to either find a way to wrap/bind it in Python to align with the description below or (preferrably) provide a pure Python implementation
   - Since sonic_y_cable (sonic-platform-common/sonic_y_cable) is already built as a package today (only for a single Y cable vendor with sonic-buildimage) vendors can also place their implementation in this directory itself.
+  - Also, a vendor can provide multiple files to support multiple cables/groups of cables. Importing the module from the mappng should import all the necessary implementation for the cable
 
 #### Proposed Solution
 
 - Vendors can place their implementations in this format
 
+```python
+	sonic_platform_common/sonic_y_cable/<vendor>/<module>
 ```
-	sonic_platform_common/sonic_y_cable/<vendor>/y_cable.py
-```
-- For example
 
+```python
+	sonic_platform_common/sonic_y_cable/<vendor>/<part_number>/<module>
+	sonic_platform_common/sonic_y_cable/<vendor>/<part_number>/<module>
 ```
+- Few examples
+
+```python
 	sonic_platform_common/sonic_y_cable/credo/y_cable.py
 ```
+
+```python
+	sonic_platform_common/sonic_y_cable/credo/PART_XYZ/y_cable_xyz.py
+	sonic_platform_common/sonic_y_cable/credo/PART_ABC/y_cable_abc.py
+	sonic_platform_common/sonic_y_cable/credo/PART_ABC/abc_helper.py
```

Comment:
Rather than having a `<part_number>` subdirectory, I think it's best to have the vendor create unique module names and keep them in the `<vendor>` directory.

Reply:
fixed